torchvision/models/detection/ssd.py

```diff
@@ -520,8 +521,34 @@ def _vgg_extractor(backbone_name: str, highres: bool, progress: bool, pretrained
 
 def ssd300_vgg16(pretrained: bool = False, progress: bool = True, num_classes: int = 91,
                  pretrained_backbone: bool = True, trainable_backbone_layers: Optional[int] = None, **kwargs: Any):
-    """
-    Constructs an SSD model with input size 300x300 and a VGG16 backbone. See `SSD` for more details.
+    """Constructs an SSD model with input size 300x300 and a VGG16 backbone.
```

Comment:
@datumbox  just FYI I copy-pasted the docs from the SSD class because we don't render the docs of the model classes yet, so the `` `SSD` `` ref wasn't resolved.

This isn't ideal and we'll need a better way of handling this in the future, but we do that for all the rest of the models for now, so it's a reasonable band aid fix. I'll bring up the discussion on how to better document the models in our next meeting

Reply:
Sounds good to me.